Sources/Card/Models/Address.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+
+/// The portable international postal address.
+/// Maps to [AddressValidationMetadata](https://github.com/googlei18n/libaddressinput/wiki/AddressValidationMetadata)
+/// and HTML 5.1 [Autofilling form controls: the autocomplete attribute](https://www.w3.org/TR/html51/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute).
```

Comment:
This comment is a little confusing to me especially the reference to HTML 5.1 and autofilling form controls. Do we need this part? Also, I wonder if we can change `The portable international postal address.` to just `The postal address of the customer.` or something like that to clarify what this is.